integration-tests/aws2/src/main/java/org/apache/camel/quarkus/component/aws/CamelRoute.java

```diff
@@ -14,21 +14,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.camel.quarkus.component.aws2.sqs.it;
+package org.apache.camel.quarkus.component.aws;
```

Comment:
Nitpick: this should better be `...aws2`

Reply:
I'm sending a PR for AWS2-S3, I'll fix this in a subsequent PR